### ISSUE TITLE
Batch_norm 替换 bug！

### ISSUE THREAD

**Issue by 2050airobert** [PaddlePaddle/PaddleDetection#7239]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

hi，
   请问下，TestReader.fuse_normalize=true 为啥我转换模型的时候不管怎么设置前面这个参数，我导出的模型都有 batch_normalxxx 这个模块呢？ 如图 
![9a822ba8c75d01b116810c2f6bcce2e](https://user-images.githubusercontent.com/8407513/199194914-c6e3a06f-d21c-4344-a302-d5d12a3c3cee.jpg)
 怎样才能在导出模型的时候去掉呢？把这个 batch norm 模块换成 下图 这两个op呢？  
![2bb29a5e2d2e07fdbf0cb3deb0fff1f](https://user-images.githubusercontent.com/8407513/199194957-fdc3dd75-35e0-43d0-923d-a695ee928342.jpg)
  期待你的答复，多谢~
BR

**Comment by jerrywgz** [comment]
fuse_normalize 做的事情只是把图像预处理部分的normalize（减均值除方差）放到网络里面，不会处理网络里面已有的结构

**Comment by 2050airobert** [comment]
那肯定啊，显而易见，所以
怎样才能实现我的这个需求呢？ 说白了就是把模型1  https://github.com/PaddlePaddle/PaddleSlim/files/9916708/tinypose_128x96.zip
变成模型2    https://github.com/PaddlePaddle/PaddleSlim/files/9915740/tinypose_act.tar.gz
需要怎么做？ 拜托大佬指点下，多谢
